packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -69,7 +69,7 @@ export default {
         },
         ratingDisplayType: {
             type: String,
-            default: null,
+            required: true,
```

Comment:
If this is required, wouldn't leaving it empty (to display no text) be impossible? Also, the README says that it's not required?

Reply:
That's a good point. Will have a think over this. 👍

Reply:
Fixed.